src/ui/pages/Menu/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import { i18n } from "../../../../../i18n";
+import { CardsPlaceholder } from "../../../../components/CardsPlaceholder";
+import "./ConnectWallet.scss";
+
+export const ConnectWallet = () => {
```

Comment:
Just to be consistent with the rest of the repo, could you please move the export to the bottom, leaving just `const` here and `export { ConnectWallet }` at the bottom of the page please.